[2025-04-27 22:07:15,410]: 47 root - INFO - Data inserted successfully to Database(MongoClient(host=['ac-zvoxsax-shard-00-02.7vp5qha.mongodb.net:27017', 'ac-zvoxsax-shard-00-01.7vp5qha.mongodb.net:27017', 'ac-zvoxsax-shard-00-00.7vp5qha.mongodb.net:27017'], document_class=dict, tz_aware=False, connect=True, retrywrites=True, w='majority', appname='Cluster0', authsource='admin', replicaset='atlas-124pw3-shard-0', ssl=True, server_api=<pymongo.server_api.ServerApi object at 0x107b99960>, ssl_ca_certs='/Users/nikhil/Documents/Project/1)Smart_Eta/myenv/lib/python3.10/site-packages/certifi/cacert.pem'), 'smarteta') database and Collection(Database(MongoClient(host=['ac-zvoxsax-shard-00-02.7vp5qha.mongodb.net:27017', 'ac-zvoxsax-shard-00-01.7vp5qha.mongodb.net:27017', 'ac-zvoxsax-shard-00-00.7vp5qha.mongodb.net:27017'], document_class=dict, tz_aware=False, connect=True, retrywrites=True, w='majority', appname='Cluster0', authsource='admin', replicaset='atlas-124pw3-shard-0', ssl=True, server_api=<pymongo.server_api.ServerApi object at 0x107b99960>, ssl_ca_certs='/Users/nikhil/Documents/Project/1)Smart_Eta/myenv/lib/python3.10/site-packages/certifi/cacert.pem'), 'smarteta'), 'eta_data') collection:
